src/distrs/gamma.jl

```diff
@@ -22,3 +25,58 @@ function gammalogpdf(k::T, θ::T, x::T) where {T<:Real}
     val = -loggamma(k) + xlogy(k - 1, xθ) - log(θ) - xθ
     return x < 0 ? oftype(val, -Inf) : val
 end
+
+gammacdf(k::T, θ::T, x::T) where {T<:Real} = first(gamma_inc(k, max(0, x)/θ))
+gammacdf(k::Real, θ::Real, x::Real)        = gammacdf(promote(float(k), θ, x)...)
+
+gammaccdf(k::T, θ::T, x::T) where {T<:Real} = last(gamma_inc(k, max(0, x)/θ))
+gammaccdf(k::Real, θ::Real, x::Real)        = gammaccdf(promote(float(k), θ, x)...)
+
+gammalogcdf(k::Real, θ::Real, x::Real) = _gammalogcdf(map(float, promote(k, θ, x))...)
```

Comment:
Again, alternatively maybe split it into two definitions?
```suggestion
gammalogcdf(k::Real, θ::Real, x::Real) = gammalogcdf(promote(k, θ, x)...)
gammalogcdf(k::T, θ::T, x::T) where {T<:Real} = _gammalogcdf(float(k), float(θ), float(x))
```